Try and exception in python
